models/models_methods_test.go

```diff
@@ -215,6 +223,14 @@ workflows:
         inputs:
         - content: echo "Hello, $NAME!"
     - bundle::print-hello:
+        inputs:
+        - NAME: Universe
+          opts:
```

Comment:
Do inputs need an opts field? I know this comes from the type so the real question is: do we want to reuse the same type?

Reply:
I tried to address this question too with the above comment: https://github.com/bitrise-io/bitrise/pull/1045#discussion_r1930664993